dotrix_math/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+pub mod math;
+pub use crate::math::slerp;
```

Comment:
Please remove `crate::` here, it is explicit, and make `math` module non-public.

Reply:
ok, actually I am not sure if I understand the difference, I will need to read about it